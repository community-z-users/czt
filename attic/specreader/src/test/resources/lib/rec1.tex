\begin{zsection}
\SECTION rec1 \parents rec2
\end{zsection}
