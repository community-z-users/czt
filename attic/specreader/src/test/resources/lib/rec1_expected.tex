net.sourceforge.czt.specreader.CyclicSectionsException: Sections are parents of each other: rec1 rec2